apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -921,7 +921,7 @@ type DatadogAgentStatus struct {
 // DatadogAgent Deployment with the Datadog Operator.
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-// +kubebuilder:unservedversion
```

Comment:
should we add this flag to v1alpha1?

Reply:
Added and adapted manifests, patches, etc. :+1: